docs/keyboardShortcuts.md

```diff
@@ -34,6 +34,7 @@ These key sequences (one key followed by the other) are used to load (in the bro
 | Filter box|<kbd>Ctrl</kbd> + <kbd>F</kbd>|Focus moves to the filter box of the selected panel|
 | Issue list|<kbd>Ctrl</kbd> + <kbd>Enter</kbd>|Focus moves to issue list of the selected panel|
 | Out of filter box (while editing filter)|<kbd>ESC</kbd> then <kbd>ESC</kbd>|First <kbd>ESC</kbd> restores previous filter and selects the filter text. The second <kbd>ESC</kbd> moves focus out of filter box|
+|n<sup>th</sup> issue (from *1* to *9*)| <kbd>Ctrl</kbd> + <kbd>n</kbd>| If the focus is already on the issue list, jumps to nth issue, where n is the issue number ranging from 1 (at the top of the list) to 9|
```

Comment:
- Second column is misleading. The `n` shouldn't be in `<kbd>`. There should be a better way to communicate that we mean literal <kbd>1</kbd> to <kbd>9</kbd> keys.
- Missing superscript in the third column